File: leetcode/1737-maximum-nesting-depth-of-the-parentheses/solution.cpp
class Solution {
public:
    int maxDepth(string s) {
        stack<int> Stack;
        int maxsize=0;
        for (char i:s){
            if(i=='('){
                Stack.push(i);
            }
            else{
                if(Stack.size()>0 && i==')'){
                Stack.pop();
            }
            }
            if(Stack.size()>maxsize){
                maxsize=Stack.size();
            }
        }
        return maxsize;
    }
};

File: leetcode/1737-maximum-nesting-depth-of-the-parentheses/solution.py
class Solution(object):
    def maxDepth(self, s):
        maxs=0
        stack=[]
        for i in s:
            if (i=='('):
                stack.append(i)
            else:
                if(len(stack)>0 and i==")"):
                    stack.pop()
            if(len(stack)>maxs):
                maxs=len(stack)
        return maxs
        """
        :type s: str
        :rtype: int
        """
        
